docs/content/contributing/governance/general-technical-review.md

```diff
@@ -0,0 +1,431 @@
+    
+    kcp can be installed on top of a Kubernetes cluster and primarily requires a mean to expose its API endpoint (e.g. load balancer support in the Kubernetes cluster). The requirements for that don't significantly change between environments. Specifically for development, the `kcp` binary has a "all-in-one" mode that makes local development against kcp possible.
+
+    For test and production environments it is strongly encouraged to run a sharded setup to validate that integrations correctly work with multiple shards.
+
+  * **Define any specific service dependencies the project relies on in the cluster.**
+
+    kcp requires [etcd](https://etcd.io) to store its data in. This can be a single etcd instance, an etcd cluster or any other project that provides an etcd-compatible API.
+
+    Installation by Helm chart has a dependency on [cert-manager](https://cert-manager.io) to manage mTLS certificates between kcp's components, but kcp itself does not rely on cert-manager, just on certificates being passed to it.
+
+  * **Describe how the project implements Identity and Access Management.**
+
+    kcp builds on top of Kubernetes' kube-apiserver code and as such, implements similar authentication and authorization methods. Specifically, kcp supports Kubernetes' Role-Based Access Control (RBAC) to assign permissions to user identities. kcp adds a few verbs and subresources to "stock" Kubernetes RBAC, which are documented [here](https://docs.kcp.io/kcp/latest/concepts/authorization/authorizers/).
+
+  * **Describe how the project has addressed sovereignty.**
+    
+    kcp can be entirely self-hosted on a Kubernetes cluster. All data is stored in an etcd instance, which can be fully managed by the installation owner. Except container images, no access to internet resources is required, and thus a kcp setup can be run fully air-gapped to address any data sovereignty concerns.
+
+  * **Describe any compliance requirements addressed by the project.**
+
+    As kcp can be run fully self-hosted, it might address any compliance requirements that require full ownership of all data. It might fulfil other compliance requirements as well, but not research has been done in this area so far.
+
+  * **Describe the project’s High Availability requirements.**
+
+    kcp requires a HA etcd cluster (so 3 or 5 etcd nodes creating a cluster) to run in HA itself. kcp scales horizontally, so multiple kcp processes pointing to the same etcd cluster can be started to accept traffic. Between those kcp processes, a leader election (via Kubernetes-style `Lease` objects) takes place to ensure that in case of a crash, another kcp process takes over active reconciliation of the kcp instance.
+
+  * **Describe the project’s resource requirements, including CPU, Network and Memory.**
+
+    A default installation from the Helm chart requires at least:
+    
+      * 1.5 cpu + 6GB RAM for three-node etcd cluster
+      * 0.1 cpu + 512MB RAM for kcp server
```

Comment:
`0.1` looks nor right. I would put at least 0.5? OR we have what to back these numbers with?

Reply:
These come from the Helm chart.